tests/integ/sagemaker/lineage/test_lineage_visualize.py

```diff
@@ -0,0 +1,274 @@
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+"""This module contains code to test SageMaker ``LineageQueryResult.visualize()``"""
+from __future__ import absolute_import
+import time
+import json
+import os
+
+import pytest
+
+import sagemaker.lineage.query
+from sagemaker.lineage.query import LineageQueryDirectionEnum
+from tests.integ.sagemaker.lineage.helpers import name, LineageResourceHelper
+
+
+def test_LineageResourceHelper(sagemaker_session):
+    # check if LineageResourceHelper works properly
+    lineage_resource_helper = LineageResourceHelper(sagemaker_session=sagemaker_session)
+    try:
+        art1 = lineage_resource_helper.create_artifact(artifact_name=name())
+        art2 = lineage_resource_helper.create_artifact(artifact_name=name())
+        lineage_resource_helper.create_association(source_arn=art1, dest_arn=art2)
+        lineage_resource_helper.clean_all()
+    except Exception as e:
+        print(e)
+        assert False
+
+
+@pytest.mark.skip("visualizer load test")
+def test_wide_graph_visualize(sagemaker_session):
+    lineage_resource_helper = LineageResourceHelper(sagemaker_session=sagemaker_session)
+    wide_graph_root_arn = lineage_resource_helper.create_artifact(artifact_name=name())
+
+    # create wide graph
+    # Artifact ----> Artifact
+    #        \ \ \-> Artifact
+    #         \ \--> Artifact
+    #          \--->  ...
+    try:
+        for i in range(10):
+            artifact_arn = lineage_resource_helper.create_artifact(artifact_name=name())
+            lineage_resource_helper.create_association(
+                source_arn=wide_graph_root_arn, dest_arn=artifact_arn
+            )
+    except Exception as e:
+        print(e)
+        lineage_resource_helper.clean_all()
+        assert False
+
+    try:
+        lq = sagemaker.lineage.query.LineageQuery(sagemaker_session)
+        lq_result = lq.query(start_arns=[wide_graph_root_arn])
+        lq_result.visualize(path="wideGraph.html")
+    except Exception as e:
+        print(e)
+        lineage_resource_helper.clean_all()
+        assert False
+
+    lineage_resource_helper.clean_all()
+
+
+@pytest.mark.skip("visualizer load test")
+def test_long_graph_visualize(sagemaker_session):
+    lineage_resource_helper = LineageResourceHelper(sagemaker_session=sagemaker_session)
+    long_graph_root_arn = lineage_resource_helper.create_artifact(artifact_name=name())
+    last_arn = long_graph_root_arn
+
+    # create long graph
+    # Artifact -> Artifact -> ... -> Artifact
+    try:
+        for i in range(10):
+            new_artifact_arn = lineage_resource_helper.create_artifact(artifact_name=name())
+            lineage_resource_helper.create_association(
+                source_arn=last_arn, dest_arn=new_artifact_arn
+            )
+            last_arn = new_artifact_arn
+    except Exception as e:
```

Comment:
it looks like this exception handling logic is similar across this function. Could we move it to one try catch block instead of repeating the logic? Also, could we use finally too? (I think the finally will still execute even if an exception is raised in the except block, but would be good to confirm)

```
try:
   . . .
except Exception as e:
  . . . 
  print(e)
  assert False
finally:
  lineage_resource_helper.clean_all()
```

This is also the case for a few of the other tests too

Reply:
`finally` always executes whether an exception occurs or not